src/app/test/dot-device.mock.ts

```diff
@@ -1,8 +1,16 @@
 import { DotDevice } from '../shared/models/dot-device/dot-device.model';
 
-export const mockDotDevice: DotDevice = {
-    cssHeight: '100',
-    cssWidth: '200',
-    name: 'iphone',
-    inode: '1'
-};
+export const mockDotDevices: DotDevice[] = [
+    {
+        cssHeight: '100',
+        cssWidth: '200',
+        name: 'iphone',
+        inode: '1'
+    },
+    {
+        cssHeight: '0',
+        cssWidth: '0',
+        name: 'bad device',
+        inode: '2'
+    }
+];
```

Comment:
shouldn't we have one with cssHeight and/or cssWidth equals to null?

Reply:
If it's null, empty string or a zero (string or number) if will be casted to 0 (number). So not sure if needed!

Reply:
What is coming from the server when those fields are empty?

Reply:
empty string

Reply:
Ok, I think we should mock that.